src/utils/components/MarkdownInput.vue

```diff
@@ -143,4 +144,13 @@ export default {
 .markdown-input-preview-card
   min-width 60%
   max-width 700px
+
+.q-field--outlined.tmp-new-border-style >>> .q-field__control:before {
+  border 1px solid rgba(0, 0, 0, 0.15)
+}
+
+.q-field--outlined.q-field--highlighted.tmp-new-border-style >>> .q-field__control:not(.text-negative):after {
+  border-color rgba(0, 0, 0, 0.45)
+}
```

Comment:
And now I see this bit is repeated a few times :) We can have style helpers, we didn't use them so much, see `src/css/` folder for some stuff. We can override quasar variables there, so if those borders were defined with variables it could be possible to do it at the quasar level.

Alternatively, perhaps defining a class, or mixin, or whatever the most appropriate stylus construct is.

(also fine to keep it like this for now, as I think that abstraction could be done more thoroughly in the future to change all forms).

Reply:
Second that. This is of course needed if we roll it out to all forms. We had something similar like `src/css/karrot.editbox.styl` in mind. Like:
- putting the css in eg `src/css/karrot.forms.styl`
- generate a webpack alias eg `forms`
- import it when needed in components:
  ```js
  <style scoped lang="stylus">
  @import '~forms'
  </style>
  ```

Or maybe do a global css so one would not need to import it all over the place.

Reply:
removed (see above). Not needed anymore.